pkg/odo/cli/component/devfile.go

```diff
@@ -266,27 +253,14 @@ func (to *TestOptions) RunTestCommand() error {
 }
 
 func warnIfURLSInvalid(url []localConfigProvider.LocalURL) {
-	// warnIfURLSInvalid checks if env.yaml contains a valid URL for the current pushtarget
+	// warnIfURLSInvalid checks if env.yaml contains a valid URL
 	// display a warning if no url(s) found for the current push target, but found url(s) for another push target
-	dockerURLExists := false
 	kubeURLExists := false
-	for _, element := range url {
-		if element.Kind == localConfigProvider.DOCKER {
-			dockerURLExists = true
-		} else {
-			kubeURLExists = true
-		}
-	}
-	var urlOutput string
-	if len(url) > 1 {
-		urlOutput = "URLs"
-	} else {
-		urlOutput = "a URL"
```

Comment:
If we care about docker based urls, this check needs to stay otherwise in case of no endpoints in a devfile, `odo push` will throw a warning

```
odo push
 ⚠  Found no valid URLs for Kubernetes

Validation
 ✓  Validating the devfile [283670ns]

Creating Kubernetes resources for component nodejs-project-htxd
 ✓  Waiting for component to start [4s]
 ✓  Waiting for component to start [1s]
```
Otherwise we can remove the entire function.

Reply:
If we are removing docker support, why would we care about docker urls? (Genuine question).

Reply:
To support the users who were using such URLs. But it was always in experimental mode though.

Reply:
Seems safe to remove to me.

Reply:
OK then we can remove this function and also the docker URL type, if it's not used anywhere anymore.

Reply:
I have removed the code that I thought was unessential. PTAL.